moveit_commander/src/moveit_commander/planning_scene_interface.py

```diff
@@ -44,8 +44,8 @@
 try:
     from pyassimp import pyassimp
 except:
-    # support pyassimp > 3.0
-    import pyassimp
```

Comment:
@davetcoleman keep original comment, and also add try - except around import, becuase if you use pyassimp 3.2 or manually fix this problem, `import pyassimp` works well.